airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/http_requester.py

```diff
@@ -421,7 +443,7 @@ def _send_with_retry(self, request: requests.PreparedRequest) -> requests.Respon
         if max_tries is not None:
             max_tries = max(0, max_tries) + 1
 
-        user_backoff_handler = user_defined_backoff_handler(max_tries=max_tries)(self._send)
+        user_backoff_handler = user_defined_backoff_handler(max_tries=max_tries)(self._send)  # type: ignore # we don't pass in kwargs to the backoff handler
```

Comment:
why did this one get necessary?

Reply:
good question. The interface of the `HttpStream._send` method is slightly different from the requester's (it accepts an extra param.

I think there are a couple of ways we can get around this
- Add an extra unused param to the requester's send_method
- Use a different backoff handler with a different callable type
- Accept that the backoff handler isn't properly typed
- Ignore the error here

None of these solutions look great to be, but I'm open to going with another one